### DISCUSSION TITLE
[Feature Request] Expose frontend interfaces/types in a npm package

### DISCUSSION THREAD

**Discussion by CoryCharlton** [home-assistant/frontend#23334] [Other feature requests]
I'm working on a custom card/view and I'm finding myself copying a lot of the frontend interfaces and types into my project. This works but it's both time consuming and can allow my copies to get out of sync with the current version.

It would really speed things up if there was a npm package that already exposed all the frontend types.

**Comment by dotlogix** [comment]
Here you go:
https://www.npmjs.com/package/home-assistant-frontend-types
